apps/tup-ui/src/pages/Dashboard/Dashboard.module.css

```diff
@@ -44,8 +44,8 @@
   display: grid;
   gap: 25px;
   grid-template-columns: 1fr;
-  grid-template-rows: 1fr 1fr;
-  grid-template-areas: 
+  grid-template-rows: auto 1fr;
+  grid-template-areas:
```

Comment:
To fix an unrelated Dashboard layout bug that I am sneaking in, because it has been a bane to the designers but I see how easy it has become to fix.